### ISSUE TITLE
Incorrect Sync

### ISSUE THREAD

**Issue by Yinka** [simon-weber/Autoplaylists-for-Google-Music#167]
I have a simple definition that returns about 130 results

![top](https://user-images.githubusercontent.com/87303/29374838-c4f5c8de-82aa-11e7-81a6-aacd6de10d87.PNG)

Autoplaylists gets the sorting wrong for most of my playlists, with this example I expected playlist that I've played 10 or 11 times to be above the ones I've played only 7 times as specified in the sort section of the definition..

See the second to the last column, you can see 7 comes before 10 and 11

![playlist](https://user-images.githubusercontent.com/87303/29374870-e159c48a-82aa-11e7-96dc-d72808fb8edd.PNG)

I've noticed this for a while, is something wrong with my query?

Thank you.

**Comment by simon-weber** [comment]
Huh, that's odd. Yeah, I'd expect the sort to work that way, too. I'll see if I can recreate it and get back to you.

**Comment by simon-weber** [comment]
Hm, I haven't been able to recreate this.

When you test the playlist do you get the incorrect ordering? Or is it only after the playlist syncs that it becomes incorrect?

**Comment by Yinka** [comment]
Yes, Test is correct, becomes incorrect after sync.
Apologies for the late reply

Thanks

**Comment by simon-weber** [comment]
Hm. My guess would be this is just Google falling behind with syncs.

Do you have the "enable change batching" setting enabled? If not, try turning it on, reloading and then waiting a few days to see if that helps.

**Comment by Yinka** [comment]
This happens even when I force-sync only that playlist, will still that be
a Google issue?

**Comment by simon-weber** [comment]
Yeah, most likely. Still you may as well trigger a sync and then [send me your logs](https://github.com/simon-weber/Autoplaylists-for-Google-Music/wiki/Debugging#viewing-your-logs) so I can check for signs of problems on our end.

**Comment by simon-weber** [comment]
Someone also brought this up on the mailing list: https://groups.google.com/d/msg/autoplaylists-for-google-music/raAoSK15Yj4/8q4h6lyxCgAJ.
